OQ_Toolkit/OQ_ARVROrigin/scripts/Locomotion_Stick.gd

```diff
@@ -17,6 +17,8 @@ onready var movement_vignette_rect = $MovementVignette_ColorRect;
 export(vr.AXIS) var move_left_right = vr.AXIS.LEFT_JOYSTICK_X;
 export(vr.AXIS) var move_forward_back = vr.AXIS.LEFT_JOYSTICK_Y;
 
+enum MovementOrientation { HEAD, HAND_LEFT, HAND_RIGHT }
+export(MovementOrientation) var movmenet_orientation := MovementOrientation.HEAD
```

Comment:
there is a typo in the variable name (and as it will be exposed in the UI I think it should be corrected).

Reply:
haha I'm a terrible speller xD I'll get it fixed.